### PR TITLE
Added some stuff to FunkinLua.hx

### PR DESCRIPTION
**Pull request by rodney528**
What I added... isPixelStage, curStage and healthPercent... the percent part says it all idk what to abbreviate it to.

### CODE REVIEW

**Comment by ImaginationSuperHero528** [comment]
And yes I change stuff around since it was originally just asked to be attached to the main branch.

**Comment by frantastic24** [comment]
getHealth already exists

**Comment by ImaginationSuperHero528** [comment]
> getHealth already exists the rest are niche additions

getHealth does not exist

**Comment by frantastic24** [comment]
My brother in Christ line 1125, use the text search feature next time

File: source/FunkinLua.hx [hunk 1]
```diff
@@ -147,6 +148,14 @@ class FunkinLua {
 		set('mustHitSection', false);
 		set('altAnim', false);
 		set('gfSection', false);
+		set('chartingMode', PlayState.chartingMode);
+
+		// Health ig
+		set('healthPercent', PlayState.instance.healthBar.percent); // Long name cause idk what to abbreviate to.
```
Comment (frantastic24) [inline source/FunkinLua.hx:154]: this is useless

**Comment by frantastic24** [comment]
healthPercent still does nothing

**Comment by ImaginationSuperHero528** [comment]
Drafting till I know how to test everything.

File: source/FunkinLua.hx (hunk 1 above)
Comment (RodneyAnImaginativePerson) [reply source/FunkinLua.hx]: Hehe no :trol:

**Comment by RodneyAnImaginativePerson** [comment]
Yeah I wanted this to die anyway.

**Pull request closed without merging**